### PR TITLE
regenerate pipeline API

### PR DESCRIPTION
**Pull request by sancyx**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
Regenrate pipeline API, to include useInstanceStore param.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

File: .gen/pipeline/version.go [hunk 1]
```diff
@@ -1,3 +1,7 @@
 package pipeline
 
+<<<<<<< HEAD
 const PipelineVersion = "0.65.0"
+=======
+const PipelineVersion = "instance-storage-api"
+>>>>>>> regenerate pipeline API
```
Comment (orymate) [inline .gen/pipeline/version.go:7]: !

**Review by orymate — CHANGES_REQUESTED** [review]

**Review by bonifaido — APPROVED** [review]

File: .gen/pipeline/version.go (hunk 1 above)
Comment (sancyx) [reply .gen/pipeline/version.go]: fixed

**Review by pregnor — APPROVED** [review]
Mergable once the final release tag is added.

File: Makefile [hunk 2]
```diff
@@ -21,7 +21,8 @@ endif
 TEST_FORMAT = short-verbose
 endif
 
-PIPELINE_VERSION = 0.65.0
+PIPELINE_VERSION = instance-storage-api
```
Comment (orymate) [inline Makefile:24]: merge only after released

**Review by orymate — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (sancyx) [reply Makefile]: yes, it was set to 0.65.1 but for some reason last commit was to picked up by the PR, anyway I've updated now.

**Review by orymate — APPROVED** [review]

**Pull request merged**